.github/workflows/ci.yml

```diff
@@ -23,3 +23,25 @@ jobs:
 
     - name: Run PHP lint
       run: composer ci:php:lint
+
+  php-code-sniffer:
+    name: PHP Code Sniffer
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        php-version:
+          - 7.2
+          - 7.3
+          - 7.4
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+
+    - name: Install Composer dependencies
+      run: composer install --prefer-dist --no-progress
```

Comment:
The prefer-dist already is configured in the `composer.json`:
https://github.com/typo3-trainer-network/tea/blob/master/composer.json#L59